### DISCUSSION TITLE
Landing page on /

### DISCUSSION THREAD

**Discussion by cinderblockgames** [alangrainger/immich-public-proxy#19] [Feature Requests]
Instead of a 404 at /, a basic landing page might be nice?

**Comment by alangrainger** [comment]
If you're sharing an album with someone, what would you want your landing page to show? Seems like anything you put there might give away information about you.

**Comment by cinderblockgames** [comment]
Just feels incomplete without something there.  Even just a centered logo or something.

**Comment by alangrainger** [comment]
> Just feels incomplete without something there. Even just a centered logo or something.

I have added this basic landing page in 1.4.5.

For those who don't want to see a landing page, you can redirect to a URL of your choice by changing your reverse proxy config, or even redirect to 404 if you like.

An example for Caddy:

```
https://immich-demo.note.sx {
    redir / https://github.com
    reverse_proxy server.lan:3000
}
```
